### PR TITLE
Added Placeholder to ChoiceSet

### LINKED ISSUE #4810: Input.ChoiceSet docs and placeholder updates
# Platform 

* Docs/website

# Details

Related #3885 

1. We don't currently document the `placeholder` property on `Input.ChoiceSet` 
2. We should also update the `value` description to explain how to use comma-separated values to specify multiple choice values

### PR DESCRIPTION
**Pull request by jwoo-msft**
## Related Issue
reference https://github.com/microsoft/AdaptiveCards/issues/3885
Fixes #4810 

- [x] Documentation
- [x] Android
- [x] iOS
- [x] UWP

## Description
Added placeholder text for Input.ChoiceSet when it is single select and compact style.

## How Verified
How you verified the fix, including one or all of the following: 
1. verified with a test card; A new test card or an updated card based on the test card is to be included with Android/WPF changes.
2. Ran unit/regression tests

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/AdaptiveCards/pull/4811)

### CODE REVIEW

File: schemas/src/elements/inputs/Input.ChoiceSet.json [hunk 1]
```diff
@@ -20,10 +20,14 @@
       "type": "string",
       "description": "The initial choice (or set of choices) that should be selected. For multi-select, specify a comma-separated string of values."
     },
+    "placeholder": {
+      "type": "string",
+      "description": "Description of the input desired. Displayed when no selection has been made, style is compact and multi selection is false."
```
Comment (matthidinger) [inline schemas/src/elements/inputs/Input.ChoiceSet.json:25]: Thanks for adding this Joseph! Can you tweak the format a bit, something like: "Description of the input desired. Only visible when no selection has been made, the `style` is `compact` and `isMultiSelect` is `false`."

**Review by matthidinger — CHANGES_REQUESTED** [review]

File: schemas/src/elements/inputs/Input.ChoiceSet.json (hunk 1 above)
Comment (jwoo-msft) [reply schemas/src/elements/inputs/Input.ChoiceSet.json:25]: sure

---
In reply to: [494550835](https://github.com/microsoft/AdaptiveCards/pull/4811#discussion_r494550835) [](ancestors = 494550835)

**Review by almedina-ms — APPROVED** [review]

**Review by almedina-ms — CHANGES_REQUESTED** [review]
You're missing changes in the .idl file, that's why the UWP build is failing

**Comment by jwoo-msft** [comment]
thanks

---
In reply to: [496028907](https://github.com/microsoft/AdaptiveCards/pull/4811#issuecomment-496028907) [](ancestors = 496028907)

**Comment by RebeccaAnne** [comment]
> {

Could we get a simple sample card that uses this? If we put it in the Elements directory and name it Input.ChoiceSet.Placeholder.json it can serve as a website sample as well.

---
Refers to: schemas/src/elements/inputs/Input.ChoiceSet.json:1 in 90f7904. [](commit_id = 90f7904cc9ca05531b5db32c212f1f65975bdcc4, deletion_comment = False)

**Review by almedina-ms — APPROVED** [review]
Please add the Input.ChoiceSet.Placeholder sample to the v1.3 folder

**Review by RebeccaAnne — APPROVED** [review]
:shipit:

**Review by matthidinger — APPROVED** [review]

**Pull request merged**